### ISSUE TITLE
screen reader cannot read the dataset attributes (L1)

### ISSUE THREAD

**Issue by JillieBeanSim** [zowe/zowe-explorer-vscode#4023]
**Describe the bug**

When using keyboard to select the 'Show Attributes' action on dataset/member, the screen reader cannot read the attribute properties of the selected dataset / member.  If it is selected by mouse (right click and select show attribute on the context menu), the screen reader can read the properties correctly.

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

- OS:
- Zowe Explorer Version:
- (Optional) Zowe CLI Version:
- (Optional) Do you have secure credentials enabled?
- (Optional) Are you using a custom credential manager?

**Additional context**
